setup.py

```diff
@@ -0,0 +1,28 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="yapapi-service-manager",
+    version="0.0.0",
+    author="Golem Factory, Jan Betley",
+    author_email="contact@golem.network, jan.betley@golem.network",
+    description="Helper tool for management of Golem-based services",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://handbook.golem.network/yapapi/",
```

Comment:
```suggestion
    url="https://github.com/golemfactory/yapapi-service-manager",
```

Reply:
The change is covered by `download_url` below I could remove... But if I get it right we foresee to integrate this library into yapapi - so the link might be correct 🤔

Reply:
OK, I don't insist.